### ISSUE TITLE
Go into project without triggering save project button to appear

### ISSUE THREAD

**Issue by hueychen27** [ScratchAddons/ScratchAddons#7549]
### Why this would be helpful

When ever I enter into my own project while it is running, it always shows the save project button. There is no point in showing the button when nothing has been edited (save can still be triggered with ctrl+s if necessary.)

### How this addon works

This addon prevents the save button from showing when the "See Inside" button is pressed in your own project. This happens only if the project is not edited.

**Comment by DNin01** [comment]
This is actually unintentional behavior caused by some addons (such as "Customizable block colors" and "Alternating nested block colors") in Scratch Addons. Try disabling the extension and the "Save Now" button will stop appearing automatically.

So, it'd be more beneficial to research what is causing this to happen and fix that issue. There may not be an easy way to work around it.

**Comment by WorldLanguages** [comment]
> This is actually unintentional behavior caused by some addons (such as "Customizable block colors" and "Alternating nested block colors") in Scratch Addons

Oh... can you create a type:bug issue?

**Comment by mxmou** [comment]
> > This is actually unintentional behavior caused by some addons (such as "Customizable block colors" and "Alternating nested block colors") in Scratch Addons
> 
> Oh... can you create a type:bug issue?

I think #2346 is about this bug, but I closed it because I was able to reproduce it without SA - I think it's related to comments attached to blocks.

**Comment by mxmou** [comment]
Apparently this always happens if SA is enabled but only in projects containing block comments if SA is disabled.

**Comment by mxmou** [comment]
I tested two different projects with and without SA - one contains block comments and one doesn't.
* When opening the editor for the project with block comments, the "Save now" button always appears, even without SA.
* If SA is disabled, opening the project without block comments never shows the save button.
* If SA is enabled, the save button appears in the project without block comments if the editor is opened from the project page. It doesn't appear if the editor is opened directly from My Stuff.

**Comment by DNin01** [comment]
@mxmou Yeah, block comments causing the "Save Now" button to show up automatically is a known thing.

You might also want to try testing with different combinations of addons enabled. Maybe start with all addons disabled and enable them one by one and see which ones make this happen.

When I tried, the addons that do this would even make the "Save Now" button appear on a dynamic enable, while the editor was open.

**Comment by mxmou** [comment]
> You might also want to try testing with different combinations of addons enabled. Maybe start with all addons disabled and enable them one by one and see which ones make this happen.

I believe it's [those that trigger workspace updates](https://github.com/search?q=repo%3AScratchAddons%2FScratchAddons%20emitworkspaceupdate&type=code).

**Comment by DNin01** [comment]
> #2346 is about this bug, but I closed it because I was able to reproduce it without SA

Should we continue this conversation in that issue? Or would it make more sense to keep this issue open and change it to https://github.com/ScratchAddons/ScratchAddons/labels/type%3A%20bug?

**Comment by mxmou** [comment]
> > #2346 is about this bug, but I closed it because I was able to reproduce it without SA
> 
> Should we continue this conversation in that issue? Or would it make more sense to keep this issue open and change it to https://github.com/ScratchAddons/ScratchAddons/labels/type%3A%20bug?

I think both can stay open - even if #2346 is fixed, an addon that fixes the vanilla bug would still be useful.

**Comment by DNin01** [comment]
Related: #2617

**Comment by hueychen27** [comment]
Save button is not triggered when going directly to editor link.
